lib/scrabble/player.rb

```diff
@@ -1,21 +1,23 @@
 module Scrabble
 	class Player
-		
+
 		attr_accessor :name, :plays
 
 		def initialize(name)
 			@name = name
 			@plays = []
+			@bag = nil
+			@current_tiles = []
 		end
 
 		def won?
-			if total_score >= 100 
+			if total_score >= 100
```

Comment:
This would be a good place to use single-line conditional like:
`(total_score >= 100): true ? false`